app/assets/javascripts/app/views/hovercard_view.js

```diff
@@ -102,14 +107,14 @@ app.views.Hovercard = app.views.Base.extend({
       if( !person || person.length === 0 ) {
         throw new Error("received data is not a person object");
       }
-
+      var personModel = new app.models.Person(person);
+      person.is_sharing = personModel.isSharing();
```

Comment:
Identifier 'is_sharing' is not in camel case.

Reply:
The handlebar helper is using this variable name, I don't think it's worth to refactor this.